python/orion_finance_sdk_py/cli.py

```
@@ -40,6 +42,17 @@
 app = typer.Typer(help="Orion Finance SDK CLI")
 
 
+def _resolve_vault(
+    config: OrionConfig, vault_address: str
+) -> OrionTransparentVault | OrionEncryptedVault:
+    """Return the SDK vault wrapper for a registered Orion vault address."""
+    if config.is_encrypted_vault(vault_address):
+        return OrionEncryptedVault()
+    if vault_address in config.orion_transparent_vaults:
+        return OrionTransparentVault()
+    raise ValueError(f"Vault address {vault_address} not in OrionConfig contract.")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)